jenkins-x.yml

```diff
@@ -107,7 +107,7 @@ pipelineConfig:
               - name: release
                 #image: docker.io/golang:1.11.5
                 # needs helm in the image for install_gitops_integration_test.go
-                image: gcr.io/jenkinsxio/builder-go:0.1.502
+                image: gcr.io/jenkinsxio/builder-go:0.1.533
```

Comment:
btw wonder if it works without a version number; then it'll pick the latest from the version stream

Reply:
good call, It should then use the platform version right?

Reply:
I've removed the version if you're able to '/lgtm', thanks @jstrachan

Reply:
hope it works now I suggested it ;) but yeah it should grab the version we maintain in the version stream: https://github.com/jenkins-x/jenkins-x-versions/blob/master/docker/gcr.io/jenkinsxio/builder-go.yml